sections/03_categorize.md

```diff
@@ -127,83 +132,88 @@ repurpose features from task to task, improving overall predictions as the field
+###### Temporal Patient Trajectories
+
+Traditionally, physician training programs justified long training hours by citing increased continuity of care and learning by following the progression of a disease over time, despite the known consequences of decreased mental and quality of life [@doi:pmid:15047076, @pmid:12691951, @pmid:2321788, @pmid:9089922]. Yet, a common practice in EHR-based research is to take a point in time snapshot and convert patient data to a traditional vector for machine learning and statistical analysis. This results in significant signal losses as timing and order of events provide insight into a patient's disease and treatment. Efforts to account for the order of events have shown promise [@doi:10.1038/ncomms5022] but require exceedingly large patient sizes due to discrete combinatorial bucketing. 
+
+Lasko et al. [@pmid:23826094] used autoencoders on longitudinal sequences of serum urine acid measurements to identify population subtypes. More recently, deep learning has shown promise working with both sequences (Convolutional Neural Networks) [@arXiv:1607.07519] and the incorporation of past and current state (Recurrent Neural Networks, Long Short Term Memory Networks)[@arXiv:602.00357v1]. 
+
+###### Data sharing and privacy
+
+Early successes using deep learning involved very large training datasets (Imagenet 1.4 million images) [@arXiv:1409.0575], but a responsibility to protect patient privacy limits the ability openly share large patient datasets. Limited dataset sizes may restrict the number of parameters that can be trained in a model, but the lack of sharing may also hamper reproducibility and confidence in results. Even without sharing data, algorithms trained on confidential patient data may present security risks or accidentally allow for the exposure of individual level patient data. Tramer et al. [@arXiv:1609.02943] showed the ability to steal trained models via public APIs and Dwork and Roth [@doi:10.1561/0400000042] demonstrate the ability to expose individual level information from accurate answers in a machine learning model.
+
+Training algorithms in a differentially private manner provides a limited guarantee that the algorithms output will be equally likely to occur regardless of the participation of any one individual. The limit is determined by a single parameter which provides a quantification of privacy. Simmons et al. [doi:doi:10.1016/j.cels.2016.04.013] present the ability to perform GWASs in a differentially private manner and Abadi et al. [arXiv:1607.00133] show the ability to train deep learning classifiers under the differential privacy framework. Finally, Continuous Analysis [doi:10.1101/056473] allows for the ability to automatically track and share intermediate results for the purposes of reproducibility without sharing the original data.
+
+
+###### Biomedical data is often "Wide"
+
+*Biomedical studies typically deal with relatively small sample sizes but each
```

Comment:
Agree this is important. This is definitely along the lines of what @xieconnect is getting at in #194 (limited standards - here you have limited examples - and maybe particularly labeled examples). Suggest again that you guys discuss and integrate this in a subsequent PR.